### DISCUSSION TITLE
Proper .wav loop points

### DISCUSSION THREAD

**Discussion by StefanH-AT** [StrataSource/Portal-2-Community-Edition#484] [Ideas]
As far as i know, to make a .wav file loop in source, you need to place markers M0 and M1. 

Wavosaur lets you set dedicated "Loop points", which source seems to ignore. Should we look into making source recognize those loop points? Are they even a thing? What's the wav file specification regarding that?

**Comment by koerismo** [comment]
I have tried using the M0/M1 markers method (I use logic, and the markers were visible in wavosaur) and it did not work for me. Only the proper loop points worked.
